test/conftest.py

```diff
@@ -37,7 +37,7 @@ def ea_params():
 
 @fixture
 def mutation_rate():
-    return 0.05
+    return 0.1
```

Comment:
why has this been changed?

Reply:
My bad, I changed this while developing and commited accidentally. it is not necessary